salt/etcd-proxy/etcd-proxy.conf.jinja

```diff
@@ -4,7 +4,7 @@
 ETCD_DATA_DIR="/var/lib/etcd/"
 ETCD_NAME="{{ grains['id'] }}"
 
-ETCD_LISTEN_CLIENT_URLS="http://0.0.0.0:2379"
+ETCD_LISTEN_CLIENT_URLS="http://127.0.0.1:2379"
 ETCD_LISTEN_PEER_URLS="https://0.0.0.0:2380"
 ETCD_ADVERTISE_CLIENT_URLS="http://{{ grains['caasp_fqdn'] }}:2379"
```

Comment:
In fact, we don't advertise IPs: we advertise names that will be resolved with the help of `/etc/hosts` in all the machines...